tries to go on even if queue manager initialization
fails. this is hand because you'll be allowed to
insert ixp400_eth immediately after you have removed
it, without re-inserting ixp400.

 ixp400_eth.c |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

--- ixp400-eth/ixp400_eth.c	1970-01-01 00:00:00.000000000 +0000
+++ ixp400-eth/ixp400_eth.c	1970-01-01 00:00:00.000000000 +0000
@@ -2850,10 +2850,7 @@ static int qmgr_init(void)
     /* Initialise Queue Manager */
     P_VERBOSE("Initialising Queue Manager...\n");
     if ((res = ixQMgrInit()))
-    {
-	P_ERROR("Error initialising queue manager!\n");
-	return -1;
-    }
+	P_ERROR("Error initialising queue manager, trying to continue!\n");
 
     TRACE;
 
